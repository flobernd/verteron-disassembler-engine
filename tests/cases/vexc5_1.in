-64 C5 8A 53 6E 2E C5 D1 14 69 7F C5 19 5D 2D 5A