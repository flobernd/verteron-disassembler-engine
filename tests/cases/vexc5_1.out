== [    BASIC ] ============================================================================================
   MNEMONIC: vrcpss [ENC: VEX, MAP: 0F, OPC: 0x53]
     LENGTH:  5
        SSZ: 64
       EOSZ: 32
       EASZ: 64
   CATEGORY: AVX
    ISA-SET: AVX
    ISA-EXT: AVX
 EXCEPTIONS: AVX5
 ATTRIBUTES: HAS_MODRM HAS_VEX 

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0   REGISTER    EXPLICIT       W     MODRM_REG    128      4      32   FLOAT32                         xmm5
 1   REGISTER    EXPLICIT       R        NDSNDD    128      4      32   FLOAT32                        xmm14
 2     MEMORY    EXPLICIT       R      MODRM_RM     32      1      32   FLOAT32  TYPE  =                 MEM
                                                                                 SEG   =                  ds
                                                                                 BASE  =                 rsi
                                                                                 INDEX =                none
                                                                                 SCALE =                   0
                                                                                 DISP  =  0x000000000000002E
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      AVX ] ============================================================================================
  VECTORLEN: 128
  BROADCAST: NONE

== [      ATT ] ============================================================================================
   ABSOLUTE: vrcpssl 0x2E(%rsi), %xmm14, %xmm5
   RELATIVE: vrcpssl 0x2E(%rsi), %xmm14, %xmm5

== [    INTEL ] ============================================================================================
   ABSOLUTE: vrcpss xmm5, xmm14, dword ptr ds:[rsi+0x2E]
   RELATIVE: vrcpss xmm5, xmm14, dword ptr ds:[rsi+0x2E]

== [ SEGMENTS ] ============================================================================================
C5 8A 53 6E 2E 
:     :  :  :..DISP
:     :  :..MODRM
:     :..OPCODE
:..VEX
